ports/ngtcp2/portfile.cmake

```diff
@@ -22,10 +22,13 @@ vcpkg_extract_source_archive_ex(
 )
 
 # Run CMake build
-set(BUILD_OPTIONS
-    # ENABLE options
-    -DENABLE_BORINGSSL=ON
-)
+if (boringssl IN_LIST FEATURES)
+    set(BUILD_OPTIONS -DENABLE_BORINGSSL=ON)
+elseif (libressl IN_LIST FEATURES)
+    set(BUILD_OPTIONS -DENABLE_OPENSSL=ON)
+else ()
+    message(FATAL_ERROR "No SSL backend specified")
+endif ()

```

Comment:
Not checking for multiple backends because both `libressl` and `boringssl` builds will fail if they see an OpenSSL header already installed.